lms/djangoapps/courseware/module_render.py

```diff
@@ -713,7 +713,6 @@ def rebind_noauth_module_to_user(module, real_user):
         wrappers=block_wrappers,
         get_real_user=user_by_anonymous_id,
         services={
-            'i18n': ModuleI18nService(),
             'fs': FSService(),
             'field-data': field_data,
             'user': DjangoXBlockUserService(user, user_is_staff=user_is_staff),
```

Comment:
@mattdrayer I was trying to get your point here but sorry I cannot get it, what is the dependency of LTI xblock on i18n service ? We are already initializing the `ModuleI18nService` in `init` method of `LmsModuleSystem` . 
[here](https://github.com/edx/edx-platform/pull/11575/files#diff-abebbc89d410d08c8dc6480fa3865847R206)
Can you please make the context simple ?